src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs

```diff
@@ -710,14 +710,10 @@ public bool Pull(string[] baseRevisions)
 				}
 				if (response.Status == PullStatus.Fail)
 				{
-					// Not sure that we need to abort the attempts just because .Net web request says so.  See Push also. CP 2012-06
-					errorMessage = "Pull data chunk failed";
-					_progress.WriteError(errorMessage);
```

Comment:
I'm interested in the cases where this is true, where do I find out more info?  TCP is a reliable transport, so the server *choosing* to send this response is interesting.